internal/action/command.go

```diff
@@ -730,6 +732,63 @@ func (h *BufPane) SetLocalCmd(args []string) {
 	}
 }
 
+// ToggleCmd toggles an option
+func (h *BufPane) ToggleCmd(args []string) {
+	if len(args) < 1 {
+		InfoBar.Error("Please provide an option to toggle")
```

Comment:
Just "Not enough arguments", for consistency with other commands (including `togglelocal`?